src/gt4py/next/embedded/nd_array_field.py

```diff
@@ -16,16 +16,16 @@
 
 import dataclasses
 import functools
-import itertools
 from collections.abc import Callable, Sequence
-from types import EllipsisType, ModuleType
-from typing import Any, ClassVar, Optional, ParamSpec, TypeAlias, TypeVar, cast, overload
+from types import ModuleType
+from typing import Any, ClassVar, Optional, ParamSpec, TypeAlias, TypeVar, overload
 
 import numpy as np
 from numpy import typing as npt
 
 from gt4py._core import definitions as core_defs
 from gt4py.next import common
+from gt4py.next.embedded import common as embedded_common
```

Comment:
Really ugly alias. Would you consider something different like  `emb_common`, `e_common`, `embedded_co`, ...?

Reply:
I don't like any including the verbose, if you want a change, you have to pick.